### ISSUE TITLE
previous output affects the cell editing

### ISSUE THREAD

**Issue by sam-s** [millejoh/emacs-ipython-notebook#269]
If the output from the previous cell has un-closed open paren, then `TAB` in the next input will indent to that previous open paren, instead of as appropriate to this cell: the `X` character is where you end up after `TAB`:
```
In [95]
print "                                      ("
                                      (
time: 2 ms


In [ ]
                                          X
                                          
```

this is because the indentation search goes beyond the cell itself.

**Comment by sam-s** [comment]
fixed by https://github.com/millejoh/emacs-ipython-notebook/pull/270

**Comment by millejoh** [comment]
Thanks @sam-s, this also fixes #252.
